projects/1.21/assets/super-factory-manager/sfm/lang/zh_cn.json

```
@@ -0,0 +1,238 @@
+  "chat.sfm.config_update_and_sync_result.internal_failure": "更新SFM配置时发生未知错误，无法确认更改是否生效。请检查服务器日志。",
+  "chat.sfm.config_update_and_sync_result.invalid_config": "提供的SFM配置无效，未进行任何更改。",
+  "chat.sfm.config_update_and_sync_result.success": "SFM配置更新成功。",
+  "container.sfm.manager": "工厂管理器",
+  "container.sfm.test_barrel_tank": "测试桶储罐",
+  "gui.jei.category.sfm.falling_anvil": "下落的铁砧",
+  "gui.jei.category.sfm.falling_anvil.consumed": "会被消耗",
+  "gui.jei.category.sfm.falling_anvil.not_consumed": "不会被消耗",
+  "gui.jei.category.sfm.printing_press": "印刷机",
+  "gui.sfm.advanced.tooltip.hint": "按住 %s 以查看详细信息。",
+  "gui.sfm.comfirm.funny.no.1": "算了，我改主意了",
+  "gui.sfm.comfirm.funny.no.2": "天呐，绝对不要",
+  "gui.sfm.comfirm.funny.no.3": "不不不不不",
+  "gui.sfm.comfirm.funny.no.4": "不了，今天不行",
+  "gui.sfm.comfirm.funny.no.5": "取消！取消！取消！",
+  "gui.sfm.comfirm.funny.no.6": "当我没说",
+  "gui.sfm.comfirm.funny.yes.1": "好吧，当然，为什么不呢。",
+  "gui.sfm.comfirm.funny.yes.2": "行，能出什么岔子呢。",
+  "gui.sfm.comfirm.funny.yes.3": "嗯，继续吧。",
+  "gui.sfm.comfirm.funny.yes.4": "尽管做。",
+  "gui.sfm.comfirm.funny.yes.5": "lol，干就完事了",
+  "gui.sfm.comfirm.funny.yes.6": "应用更改。",
+  "gui.sfm.container_inspector.container_slot_count": "容器槽位：%d",
+  "gui.sfm.container_inspector.inventory_slot_count": "物品栏槽位：%d",
+  "gui.sfm.container_inspector.mekanism_machine_inputs": "以下基于机器的输入配置",
+  "gui.sfm.container_inspector.mekanism_machine_outputs": "以下基于机器的输出配置",
+  "gui.sfm.container_inspector.mekanism_null_direction_warning": "通用机械模组方块在未设置方向时仅可读！！！！",
+  "gui.sfm.container_inspector.notice.1": "GUI槽位不一定总是对应自动化槽位！！！",
+  "gui.sfm.container_inspector.notice.2": "按 %s 以切换此覆盖层。",
+  "gui.sfm.container_inspector.show_exports_button": "导出检查器",
+  "gui.sfm.disk.tooltip.edit_in_hand": "手持磁盘时右击可直接编辑，无需管理器。",
+  "gui.sfm.exit_without_saving_confirm.message": "确定要放弃当前更改吗？",
+  "gui.sfm.exit_without_saving_confirm.no_button": "继续编辑",
+  "gui.sfm.exit_without_saving_confirm.title": "不保存直接退出？",
+  "gui.sfm.exit_without_saving_confirm.yes_button": "不保存退出",
+  "gui.sfm.facade_confirm_apply.message": "%d 种伪装状态将覆盖 %d 个方块。",
+  "gui.sfm.facade_confirm_apply.title": "确定要更新伪装外观吗？",
+  "gui.sfm.facade_confirm_change_world_block.message": "%d 个方块将改变形状，其伪装会被保留。",
+  "gui.sfm.facade_confirm_change_world_block.title": "确定要更改伪装的世界方块吗？",
+  "gui.sfm.facade_confirm_clear.message": "%d 种伪装状态将从 %d 个方块上清除。",
+  "gui.sfm.facade_confirm_clear.title": "确定要清除这些伪装吗？",
+  "gui.sfm.item_inspector.copied_to_clipboard": "已复制 {} 字符到剪贴板！",
+  "gui.sfm.label_gun.button.toggle_label_view": "循环标签视界",
+  "gui.sfm.label_gun.clear_button": "清除",
+  "gui.sfm.label_gun.label_button": "%s (%d)",
+  "gui.sfm.label_gun.label_edit_placeholder": "搜索或输入新标签",
+  "gui.sfm.label_gun.placeholder": "标签",
+  "gui.sfm.label_gun.prune_button": "清除冗余",
+  "gui.sfm.logs.button.clear_logs": "清空日志",
+  "gui.sfm.logs.button.clear_logs.packet_received": "日志已清空",
+  "gui.sfm.logs.button.copy_logs": "复制日志",
+  "gui.sfm.logs.button.copy_logs.tooltip": "Shift + 点击 以复制原始数据",
```

| **审查级别** |
|:-:|
| ℹ️ 信息 / 不阻止合并  |

**原文: Shift-click for raw**

```suggestion
  "gui.sfm.logs.button.copy_logs.tooltip": "按住 Shift 点击以复制原始数据",
```